hack/make/binary

```diff
@@ -20,21 +20,30 @@ if [ "$(go env GOOS)/$(go env GOARCH)" != "$(go env GOHOSTOS)/$(go env GOHOSTARC
 	esac
 fi
 
-if [ "$(go env GOOS)" == "linux" ] ; then
-	case "$(go env GOARCH)" in
-		arm*|386)
-			# linking for Linux on arm or x86 needs external linking to avoid
-			# https://github.com/golang/go/issues/9510 until we move to Go 1.6
-			if [ "$IAMSTATIC" == "true" ] ; then
-				export EXTLDFLAGS_STATIC="$EXTLDFLAGS_STATIC -zmuldefs"
-				export LDFLAGS_STATIC_DOCKER="$LDFLAGS_STATIC -extldflags \"$EXTLDFLAGS_STATIC\""
+case "$(go env GOOS)" in
+	linux)
+		case "$(go env GOARCH)" in
+			arm*|386)
+				# linking for Linux on arm or x86 needs external linking to avoid
+				# https://github.com/golang/go/issues/9510 until we move to Go 1.6
+				if [ "$IAMSTATIC" == "true" ] ; then
+					export EXTLDFLAGS_STATIC="$EXTLDFLAGS_STATIC -zmuldefs"
+					export LDFLAGS_STATIC_DOCKER="$LDFLAGS_STATIC -extldflags \"$EXTLDFLAGS_STATIC\""
 
-			else
-				export LDFLAGS="$LDFLAGS -extldflags -zmuldefs"
-			fi
-			;;
-	esac
-fi
+				else
+					export LDFLAGS="$LDFLAGS -extldflags -zmuldefs"
+				fi
+				;;
+		esac
+		;;
+	windows)
+		# !!! TEMPORARY HACK !!!
+		# See Dockerfile
+		if [ "$(go env GOARCH)" == "amd64" ]; then
```

Comment:
Ah, this is going to fail win2lin and windowsTP5 which both build a binary locally which isn't cross compiled. This needs some extra logic so that it only happens on the cross-compile scenario targetting windows/amd64